components/tikv_util/src/logger/file_log.rs

```diff
@@ -83,6 +94,9 @@ impl RotatingFileLoggerBuilder {
 
     pub fn build(mut self) -> io::Result<RotatingFileLogger> {
         let file = open_log_file(&self.path)?;
+        let mut worker = LazyWorker::new("archive-worker");
+        assert!(worker.start(Runner::new(&self.path, self.max_backups, self.max_days)));
+        worker.scheduler().schedule(Task::Archive).unwrap();
```

Comment:
Why schedule a task here?

Reply:
I think there may be such a scene:

User previously set not to archive. After a period of time, the user found that the number of rotated files was too many, so they changed the archiving rules and restarted TiKV. If we don't schedule a task here, we can only perform the archiving operation when the rotate is triggered next time, which may be a bit long.